themes/classic/assets/js/theme.js

```diff
@@ -9191,7 +9191,7 @@
 /***/ function(module, exports, __webpack_require__) {
 
 	var __WEBPACK_AMD_DEFINE_FACTORY__, __WEBPACK_AMD_DEFINE_ARRAY__, __WEBPACK_AMD_DEFINE_RESULT__;/*!
-	 * Datepicker for Bootstrap v1.6.0 (https://github.com/eternicode/bootstrap-datepicker)
+	 * Datepicker for Bootstrap v1.5.1 (https://github.com/eternicode/bootstrap-datepicker)
```

Comment:
This change doesn't seem related to the commit, I don't see anything about the datepicker in the source files. Maybe someone forgot to re-build the assets in a parent commit.